Gemfile

```diff
@@ -30,6 +29,7 @@ group :development, :test do
 end
 
 group :development do
+  gem "decidim-dev", DECIDIM_VERSION
```

Comment:
I think this might be needed in production to run the seeds?

Reply:
It doesn't seem to be necessary! https://decidim-staging-pr-266.herokuapp.com